extensions/2.0/Khronos/KHR_materials_transmission/schema/glTF.KHR_materials_transmission.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_transmission glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the optical transmission of a material.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "transmissionFactor": {
+            "type": "number",
+            "description": "The base percentage of light transmitted through the surface.",
+            "default": 0.0,
+            "minimum": 0.0,
+            "maximum": 1.0,
+            "gltf_detailedDescription": "The base percentage of non-specularly reflected light that is transmitted through the surface. i.e. of the light that penetrates a surface (isn't specularly reflected), this is the percentage that is transmitted and not diffusely re-emitted."
+        },
+        "transmissionTexture": {
+            "allOf": [ { "$ref": "textureInfo.schema.json" } ],
+            "description": "A greyscale texture that defines the transmission percentage of the surface. This will be multiplied by transmissionFactor.",
+            "gltf_detailedDescription": "The percentage of non-specularly reflected light that is transmitted through the surface. i.e. of the light that penetrates a surface (isn't specularly reflected), this is the percentage is transmitted and not diffusely re-emitted. This will be multiplied by the transmissionFactor."
```

Comment:
Please sync the descriptions here with the README (wrt use of red channel and linear transfer function).

Reply:
@lexaknyazev I pushed a change here, can you check it?  Thanks!

Reply:
LGTM for now. We should probably do another pass on all PBR Next extensions to align the language with the core spec as we did in #1606.